### ISSUE TITLE
This board sucks, No compile for SKR mini E3 V2.0[BUG] (short description)

### ISSUE THREAD

**Issue by dodgedoc** [bigtreetech/BIGTREETECH-SKR-mini-E3#377]
BTT  You release a board that does not even compile under VSCode.  What?????  I need help with this or you can come get your board and screens I have got.  I can download code for TH3D Easyboard and it complies fine.. What is up with you guys.. If it in platformIO then update it.. This is letting ppl down.. 
This is just my thoughts... Barry

**Comment by casterle** [comment]
I just noticed I can no longer build the code I downloaded today with VSCode either. The previous version did build so you might try that.

**Comment by tablatronix** [comment]
Whats the error? I compile under pio fine.

**Comment by pbsuper** [comment]
if you just get a fresh firmware file from marlin and set it up for this board so the right CPU and such it does compile but the BTT Tree just sucks A.T.M.

**Comment by tablatronix** [comment]
oh the repo has problems?

**Comment by pbsuper** [comment]
Yes they are using the wrong lib for neopixel with this cpu so. You can't compile it cause of that

**Comment by tablatronix** [comment]
Good to know , I am about to add my neopixels tonight

**Comment by dodgedoc** [comment]
I got a file from CMendy, It would not compile either. The under platformIO, i found under quick access that you can update all libraries so i updated them. Not many out of date. Tried to compile after that, no go. Back to quick access, found upgrade platformIO core. I upgrade the core? dont know what it is. Then i tried to compile after upgrading. Guess what it worked.
But will not compile with the file from BTT. 
Hope that helps someone out.. 
Barry

**Comment by tablatronix** [comment]
I did just see a neopixel issue earlier when I was browsing closed issues

**Comment by btolarz** [comment]
As workaround copy https://github.com/MarlinFirmware/Marlin/tree/2.0.x/buildroot and then compile. Last commit on this repo broke everything.
